src/main/java/com/fundation/webservice/model/CriteriaAudio.java

```diff
@@ -17,48 +17,60 @@
  */
 public class CriteriaAudio extends CriteriaConvert {
     private String newFormat;
-    private String aCodec;
-    private int aBit;
-    private int aChannel;
-    private int aRate;
+    private String audioCodec;
+    private int audioBit;
+    private int audioChannel;
+    private int audioRate;
 
+    /**
+     *
+     * Method getNewFormat receive the new format audio.
+     */
```

Comment:
Single line comment please.

Reply:
The comment in third person please. You can review examples in the class "CriteriaThumbnailVideo.java".

Reply:
this documentation can review later please.